Sources/JExtractSwiftLib/FFM/FFMSwift2JavaGenerator+JavaBindingsPrinting.swift

```diff
@@ -387,16 +386,29 @@ extension FFMSwift2JavaGenerator {
   }
 
   private func printDeclDocumentation(_ printer: inout CodePrinter, _ decl: ImportedFunc) {
-    printer.print(
-      """
-      /**
-       * Downcall to Swift:
-       * {@snippet lang=swift :
-       * \(decl.signatureString)
-       * }
-       */
-      """
-    )
+    if var documentation = SwiftDocumentationParser.parse(decl.swiftDecl) {
+      if let translatedDecl = translatedDecl(for: decl), translatedDecl.translatedSignature.requiresSwiftArena {
+        documentation.parameters.append(
+          SwiftDocumentation.Parameter(
+            name: "swiftArena$",
+            description: "the arena that will manage the lifetime and allocation of Swift objects"
+          )
+        )
+      }
+
+      documentation.print(in: &printer)
+    } else {
+      printer.print(
+        """
+        /**
+         * Downcall to Swift:
+         * {@snippet lang=swift :
+         * \(decl.signatureString)
+         * }
+         */
+        """
```

Comment:
I think we likely want to keep the "downcall to ..." part of the docs always; it's a quick way to verify if this is calling what you think it is, and if we have any bad codegen etc

Reply:
In other words, I don't think there should be an if/else here, can we centralize "make the docs, as good as possible" in some docs type that may use the parser inside?